# $DragonFly: src/sys/dev/acpica5/debugger,dbstats.c.patch,v 1.1 2007/01/25 15:12:06 y0netan1 Exp $

--- dbstats.c.orig	2007-01-18 01:29:39.000000000 +0900
+++ dbstats.c	2007-01-25 12:02:30.000000000 +0900
@@ -141,10 +141,6 @@
     void                    *Context,
     void                    **ReturnValue);
 
-static void
-AcpiDbListInfo (
-    ACPI_MEMORY_LIST        *List);
-
 
 /*
  * Statistics subcommands
@@ -171,6 +167,9 @@
 
 
 #if defined ACPI_DBG_TRACK_ALLOCATIONS || defined ACPI_USE_LOCAL_CACHE
+static void
+AcpiDbListInfo (
+    ACPI_MEMORY_LIST        *List);
 
 /*******************************************************************************
  *
